pubtools/pulplib/_impl/client/client.py

```diff
@@ -203,6 +203,33 @@ def search_repository(self, criteria=None):
             Repository, "repositories", criteria=criteria, search_options=search_options
         )
 
+    def search_units_by_type(self, content_type, criteria=None):
+        """Search for units of single type across all repositories.
+
+        Args:
+            criteria (:class:`~pubtools.pulplib.Criteria`)
+                A criteria object used for this search.
+                If None, search for all repositories.
```

Comment:
This doc string doesn't seem right.  Copy-pasted?  Aren't we *always* searching over all repositories, and this criteria is applied on units, not repositories?